docs/how-to/rocm-for-hpc/index.rst

```diff
@@ -0,0 +1,239 @@
+applications.
+
+* For more information, see :doc:`What is ROCm? <../../what-is-rocm>`.
+
+* For guidance on installing ROCm, see :doc:`rocm-install-on-linux:index`. See
+  the :doc:`../../compatibility/compatibility-matrix` for details on hardware
+  and operating system support.
+
+Some of the most popular HPC frameworks are part of the ROCm platform, including
+those to help parallelize operations across multiple accelerators and servers,
+handle memory hierarchies, and solve linear systems.
+
+.. image:: ../../data/how-to/rocm-for-hpc/hpc-stack-2024_6_20.png
+   :align: center
+   :alt: Software and hardware ecosystem surrounding ROCm and AMD Instinct for HPC
+
+The following catalog of GPU-accelerated solutions includes a vast set of
+platform-compatible HPC applications, including those for astrophysics, climate 
+and weather, computational chemistry, computational fluid dynamics, earth
+science, genomics, geophysics, molecular dynamics, and physics computing. Refer
+to the resources in the following table for instructions on building, running,
+and deployment using AMD Instinct accelerators.
+
+.. _hpc-apps:
+
+..
+   Reduce font size of HPC app descriptions slightly.
+
+.. raw:: html
+
+   <style>
+     #hpc-apps-table tr td:last-child {
+       font-size: 0.9rem;
+     }
+   </style>
+
+.. container::
+   :name: hpc-apps-table
+
+   .. list-table::
+      :header-rows: 1
+      :stub-columns: 1
+      :widths: 2 2 5
+
+      * - Application domain
+        - HPC application
+        - Description
+
+      * - Physics
+        - `Chroma <https://github.com/amd/InfinityHub-CI/tree/main/chroma/>`_
+        - Data-parallel programming constructs for lattice field theory -- lattice
+          quantum chromodynamics (QCD) in particular. It's used to simulate the
+          behavior of quarks and gluons. Chroma uses SciDAC QDP++ data-parallel
+          programming (in C++) that presents a single high-level code image to
+          the user, but can generate highly optimized code for many
+          architectural systems including single node workstations, multi and
+          many-core nodes, clusters of nodes via QMP, and classic vector
+          computers.
+
+      * -
+        - `Grid <https://github.com/amd/InfinityHub-CI/tree/main/grid/>`_
+        - Library for lattice QCD calculations that uses a high-level data
+          parallel approach and techniques to target multiple types of
+          parallelism. Grid currently supports MPI, OpenMP and short vector
+          parallelism. The code requires at least one AMD accelerator to run.
+
+      * -
+        - `MILC <https://github.com/amd/InfinityHub-CI/tree/main/milc/>`_
+        - Set of research codes developed by the MIMD Lattice Computation (MILC)
+          collaboration for doing simulations of four-dimensional SU(3) lattice
+          gauge theory on MIMD parallel machines scaling from single-processor
+          workstations to HPC systems.
```

Comment:
The MIMD Lattice Computation (MILC) collaboration developed a set of research codes for simulations of four-dimensional SU(3) lattice gauge theory on MIMD parallel machines scaling from single-processor workstations to HPC systems.